valkyrie/templates/deployment.yaml

```diff
@@ -23,6 +23,13 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          livenessProbe:
+            exec:
+              command:
+                - /app/bin/valkyrie
```

Comment:
so does this _not_ need to implement the ulimit controls to ensure it doesn't eat a ton of resoures?

Reply:
that will probably be the next step using softlimit and discuss it on entry to see if its scope for this card. We are not certain yet that performing the ping will completely address the issue `erl_child_setup closed` issue

Reply:
https://rhye.org/post/erlang-kubernetes-liveness/